src/atoms/Navbar/Navbar.js

```diff
@@ -255,6 +268,7 @@ class Navbar extends Component {
 		const toggleMenuClass = this.state.showMenu
 			? cx(toggleMenu.className, 'active')
 			: toggleMenu.className;
+		const isNavbarList = this.checkListComponent();
```

Comment:
use getter wherever is possible so the code looks cleaner, I think a getter property named `this.isNavbarList` will do the same job.

Reply:
Yeah makes sense I will change this.